docroot/profiles/custom/cgov_site/modules/custom/cgov_site_section/cgov_site_section.module

```diff
@@ -55,10 +55,10 @@ function cgov_site_section_taxonomy_term_presave(EntityInterface $entity) {
         $values[] = $value;
       }
       $computed_parent = implode('/', $values);
-      $entity->set('computed_path', $computed_parent . '/' . $field_pretty_url);
+      $entity->set('computed_path', '/' . $computed_parent . '/' . $field_pretty_url);
```

Comment:
So I am just going to point out that this should be handled with an implode..., but I will approve